unittests/allplatformstests.py

```diff
@@ -3076,6 +3076,11 @@ def test_alias_target(self):
         self.run_target('build-all')
         self.assertPathExists(os.path.join(self.builddir, 'prog' + exe_suffix))
         self.assertPathExists(os.path.join(self.builddir, 'hello.txt'))
+        expected = 'a run target was here'
+        out = self.run_target('aliased-run')
+        out_start = out.find(expected)
+        out_line = out[out_start:].rstrip('\n')
+        self.assertEqual(expected, out_line)
```

Comment:
`self.assertIn(b'a run target was here', out)` should be enough.

Reply:
Could also self.assertNotIn() when building `build-all`.

Reply:
That one is particular, the summary text is made with dictionaries that does not guarantee ordering with python 3.6. So some lines of that output can be reordered depending on you python version.

Reply:
Ah, I was wondering if maybe it was trying to be strict about like, whole-line searches or something.